androidApp/build.gradle.kts

```diff
@@ -60,11 +59,11 @@ android {
     compileOptions {
         // Flag to enable support for the new language APIs
         isCoreLibraryDesugaringEnabled = true
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
```

Comment:
why did you update java version? it is required for old android support

Reply:
Ah! I got an IDE warning. Have reverted, but I think soon we could update to 11?